srk/src/terminationLLRF.ml

```diff
@@ -0,0 +1,425 @@
+open Syntax
+module Vec = Linear.QQVector
+module Mat = Linear.QQMatrix
+module TF = TransitionFormula
+
+include Log.Make(struct let name = "TerminationLLRF" end)
+
+type analysis_result = ProvedToTerminate | Unknown
```

Comment:
isomorphic to bool -- just return true or false instead.

Reply:
I thought the current version is a bit more intuitive to understand.